common-content/en/module/tools/ls/index.md

```diff
@@ -0,0 +1,21 @@
++++
+title = "ls"
+headless = true
+time = 15
+facilitation = false
+emoji= "💻"
+[objectives]
+    1="List the files (and directories) in a directory with ls"
++++
+
+`ls` is a tool for listing the files in a directory.
+
+Learn about it from its man page (and the backlog exercises).
+
+Some commonly used flags you should understand:
```

Comment:
Be more directive here. Could you set a task to write them out? Or include some multiple choice quizzes?

Try to include one action per block of info where they can stop and check for understanding. Motivating the attention is the goal here -- help the learner not to zone out and click through mindlessly.

Reply:
I have given a _few_ example exercises for each block, but not exhaustively for every flag mentioned here.